examples/minetest/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# build
+FROM golang:1.14.12 as builder
+
+WORKDIR /go/src/minetest
+
+COPY main.go .
+RUN go get agones.dev/agones/pkg/sdk \
+    && go build -o wrapper .
+
+# final image
+FROM debian:buster-backports
```

Comment:
Buster is not the current version. Please use a current version of debian to reduce the changes of security issues.

Reply:
Changed to latest alpine to minimize security issues.